melodies_monet/driver.py

```diff
@@ -1364,10 +1360,11 @@ def pair_data(self, time_interval=None):
                             
                             paired_data = sutil.omps_nm_pairing_apriori(model_obj,obs.obj,keys)
                         else:
-                            model_obj = mod.obj[keys+['dp_pa']]
+                            keys.extend('dp_pa')
+                            model_obj = mod.obj[keys]
                             paired_data = sutil.omps_nm_pairing(model_obj,obs.obj,keys)
```

Comment:
'dp_pa' is added within this function too, maybe remove that. Or revert your change here.

Reply:
And it looks like the function is expecting a single string to be passed, based on the arg name `ozone_ppbv_varname` and `pair_variables = ['dp_pa', ozone_ppbv_varname]`. Some fixes are needed (decide whether to pass single string var or list of strings, and adjust the function accordingly).